keyvalue/clients/src/main/java/tachyon/client/keyvalue/KeyValueIterator.java

```diff
@@ -31,10 +31,11 @@
   boolean hasNext();
 
   /**
+   * Throws a {@link java.util.NoSuchElementException} if there are no more pairs.
```

Comment:
@apc999 should we instead of this change add NoSuchElement to the list of exceptions that can be thrown?